src/overlays/actors/ovl_En_Horse_Link_Child/z_en_horse_link_child.c

```diff
@@ -73,41 +88,59 @@ static ColliderJntSphInit sJntSphInit = {
 
 static CollisionCheckInfoInit sColChkInfoInit = { 10, 35, 100, MASS_HEAVY };
 
-s32 D_808DFF10[] = { 1, 19 };
+// the animation frames that should have sfx
+static s32 sAnimSoundFrames[] = { 1, 19 };
 
-f32 D_808DFF18[] = { 1.0f, 1.0f, 1.5f, 1.5f, 1.5f };
+static f32 sAnimPlaySpeeds[] = { 1.0f, 1.0f, 1.5f, 1.5f, 1.5f };
```

Comment:
```suggestion
static f32 sAnimPlaySpeeds[OOT_CHILD_EPONA_ANIM_MAX] = { 
1.0f, // OOT_CHILD_EPONA_ANIM_IDLE
1.0f, // OOT_CHILD_EPONA_ANIM_WHINNY
1.5f, 
1.5f, 
1.5f, 
};
```
And fill in the rest